tests/framework/common_apps.go

```diff
@@ -273,6 +273,17 @@ func (td *OsmTestData) SimplePodApp(def SimplePodAppDef) (corev1.ServiceAccount,
 							corev1.ResourceMemory: resource.MustParse("128Mi"),
 						},
 					},
+					Env: []corev1.EnvVar{
+						{
+							Name: "POD_IP",
+							ValueFrom: &corev1.EnvVarSource{
+								FieldRef: &corev1.ObjectFieldSelector{
+									APIVersion: "v1",
+									FieldPath:  "status.podIP",
+								},
+							},
+						},
+					},
```

Comment:
Why is this needed when the init-container adds this during sidecar injection?

Reply:
This is so that the test httpbin pod can bind to the pod IP